models/zendesk__ticket_field_history.sql

```diff
@@ -14,7 +14,7 @@ with change_data as (
     from {{ ref('int_zendesk__field_history_scd') }}
   
     {% if is_incremental() %}
-    where valid_from >= (select max(date_day) from {{ this }})
+    where valid_to >= (select max(date_day) from {{ this }})
```

Comment:
I actually did more digging on this and we do in fact want to retain this as `valid_from` as this will be able to capture updates. If there are no updates on a ticket then we will not want to bring the tickets in on this cte. That is why the `most_recent_change` cte is used.